packages/terra-paginator/tests/jest/ControlledPaginator.test.jsx

```diff
@@ -7,24 +7,38 @@ import ControlledPaginator from '../../src/ControlledPaginator';
 
 describe('Paginator', () => {
   const defaultRender = <ControlledPaginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} />;
+  const renderWithHeadingId = <ControlledPaginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} paginatorHeadingId="paginator_header_id" />;
+  const renderWithAriaLabel = <ControlledPaginator onPageChange={e => typeof e} selectedPage={1} totalCount={2234} itemCountPerPage={20} paginatorLabel="paginator_label" />;
 
   // Snapshot Tests
+  it('should render a Controlled Paginator with heading id set - Tiny', () => {
+    const wrapper = shallowWithIntl(renderWithHeadingId).dive();
+    wrapper.setState({ breakpoint: 'tiny' });
+    wrapper.update();
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render a Controlled Paginator with aria label set - Tiny', () => {
+    const wrapper = shallowWithIntl(renderWithAriaLabel).dive();
+    wrapper.setState({ breakpoint: 'tiny' });
+    wrapper.update();
+    expect(wrapper).toMatchSnapshot();
```

Comment:
We should test for the aria label as well

Reply:
Added: https://github.com/cerner/terra-core/pull/3867/commits/450ee57249ee65b6a8508c71973f7b2373d577ae#diff-e05da9d01a4643a4ec7a339a471781e030e36c6b9f8828137fe97345b6934c13R42-R50